modules/core/src/main/java/com/illposed/osc/transport/OSCPortInBuilder.java

```diff
@@ -97,6 +121,11 @@ public OSCPortInBuilder setNetworkProtocol(final NetworkProtocol protocol) {
 		return this;
 	}
 
+	public OSCPortInBuilder setTransport(final Transport networkTransport) {
+		transport = networkTransport;
```

Comment:
No must, and I see it is the same in the other methods, which is probably why you did it this way, but I' rather do this as `this.transport = transport;`, vs choosing arbitrary different names for the two variables.
As said.. leaving it this way is totally fine too.